script.js

```diff
@@ -82,20 +88,22 @@ function checkWinner(gameState) {
     ) {
       winMessage = "Player 2 wins!";
     }
-  });
+
+  })
 
   return winMessage;
 }
 
 function playGame() {
-  const tiles = document.querySelectorAll(".tile");
-
-  startBtn.removeEventListener("click", playGame, false);
+  startBtn.removeEventListener('click', playGame, false);
   startBtn.style.opacity = 0.5;
 
-  tiles.forEach(function (tile) {
-    tile.addEventListener("click", clickEvent);
-  });
+  tiles.forEach((tile) => {
```

Comment:
is this the part that was similar to ours? forgot to say we had a cls of the clickedX and clickedO but works the same i htink?